Bottom: 61d67da51d9155849464fd52ecce54d5d255a81c
Top:    f051da871c0d4cb75f558cde940bdf7846570f36
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 04:19:55 +0000

HMM - THIS APPEARS TO FIX BROKEN DRILLED HOLES IN THE PCB CORE


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f1a4a67..0d97f8f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -300,7 +300,7 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
       *cx = COORD_TO_STEP_X (PCB, contour->cx);
       *cy = COORD_TO_STEP_Y (PCB, contour->cy);
       *r = COORD_TO_MM (contour->radius);
-      *cw = (contour->Flags.orient == PLF_DIR);
+      *cw = (contour->Flags.orient != PLF_DIR);
       return;
     }
 #endif
